src/test/java/com/ot/restaurant/entities/IoCContainerTest.java

```diff
@@ -0,0 +1,107 @@
+package com.ot.restaurant.entities;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class IoCContainerTest {
+
+  @Test
+  void whenInstantiatingIoCContainerShouldInstanceTheSame() {
+
+    IoCContainer ioc = IoCContainer.getInstance();
+    IoCContainer ioc2 = IoCContainer.getInstance();
+
+    assertEquals(ioc, ioc2);
+  }
+
+  @Test
+  void whenRegisterValueShouldResolveValueThenAssertSucceeds() throws Exception {
+    IoCContainer ioc = IoCContainer.getInstance();
+
+    String key = "keyEntiti";
+    Object value = new Object();
```

Comment:
Could you use the test setup to always setup a:
1. ioc
2. key
3. value

?

Reply:
Yes, Done